commands/view-state.js

```diff
@@ -22,12 +21,26 @@ module.exports = {
                 '`final` is for a block that has been validated on at least 66% of the nodes in the network',
             type: 'string',
             choices: ['optimistic', 'final'],
-            
         })
         .option('utf8', {
             desc: 'Decode keys and values as UTF-8 strings',
             type: 'boolean',
             default: false
+        })
+        .conflicts('block-id', 'finality')
+        .check((argv) => {
+            if (!argv.finality && !argv.blockId) {
+                throw new Error('Must provide either --finality or --blockId');
+            }
+            return true;
+        })
+        .coerce({
+            blockId: (blockId) => {
```

Comment:
:nit: Not a blocker, but this function can be co-located with the `block-id` option definition (unlike `conflicts` which must be globally defined) -- `coerce` is a valid property that you can provide to the `.option()` call along with stuff like `desc`, `type`, `default`, `required`, etc. :)